docs/examples/cognito/basic_usage.rst

```diff
@@ -0,0 +1,11 @@
+Misc
+-------------------
+
+Also, it works with other AWS services as well.
```

Comment:
These sounds very generic to me. Take a look at docs/examples/sqs/queue.rst and try to make the text more relevant.